frontend/src/App.jsx

```diff
@@ -20,10 +21,10 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/login" element={<Login />} />
         <Route path="/myExperiences" element={<MyExperiences />} />
-        <Route path="/createExperience" element={<CreateExperience />} />
-        <Route path="/myTrips" element={<MyTrips />} />
-        <Route path="/individualTrip" element={<IndividualTrip />} />
-        <Route path="/editTrip" element={<EditTrip />} />
+        <Route path="/createExperience" element={<CreateExperience />} /> 
+        <Route path="/myTrips" element={<MyTrips userId={123}/>} />       
```

Comment:
Saw your comment about replacing the hardcoded id. Once I get the logic working for user login sessions I'll reach out to you and we can get this adjusted!